statistics/handle/handle.go

```diff
@@ -1012,7 +1022,6 @@ func (h *Handle) statsCacheLen() int {
 // Callers should add retry loop if necessary.
 func (h *Handle) updateStatsCache(newCache *cache.StatsCache) (updated bool) {
 	h.statsCache.Replace(newCache)
-	handle_metrics.CostGauge.Set(float64(newCache.Cost()))
```

Comment:
Does this mean now we no longer update the metrics when the stats cache updates?

Reply:
we do not use to update the stats cache to replace the old with the old one. we just set the cache by mutex.

Reply:
It mean current actual size.